scripts/generate_build_and_tag.sh

```diff
@@ -0,0 +1,31 @@
+#!/bin/bash
+
+# This file is licensed under the Apache License v2.0 with LLVM Exceptions.
+# See https://llvm.org/LICENSE.txt for license information.
+# SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+
+# Updates for the current LLVM commit and creates a tag.
+#  1. Generates a new LLVM build file for the current LLVM commit.
+#  2. If there are diffs (including if the current LLVM submocule change was not
+#     committed), creates a new commit.
+#  3. Tags the HEAD commit with the matching LLVM commit.
+
+set -e
+set -o pipefail
+
+SUBMODULE_DIR="third_party/llvm-project"
+
+LLVM_COMMIT="$(git submodule status -- ${SUBMODULE_DIR?} | awk '{print $1}' | tr -d '+')"
+
+sed -i '/Begin autogenerated content/Q' llvm-bazel/llvm-project-overlay/llvm/BUILD
+python3 ./llvm-bazel/generate_bazel_build.py \
+  --llvm_root=third_party/llvm-project/llvm \
```

Comment:
I think you want to use the SUBMODULE_DIR here.

Reply:
Done, thanks for catching